src/main/java/de/muenchen/isi/configuration/ImportStammdatenFromFile.java

```diff
@@ -0,0 +1,158 @@
+package de.muenchen.isi.configuration;
+
+import de.muenchen.isi.domain.exception.CsvAttributeErrorException;
+import de.muenchen.isi.domain.exception.FileImportFailedException;
+import de.muenchen.isi.domain.service.stammdaten.StammdatenImportService;
+import de.muenchen.isi.infrastructure.repository.stammdaten.SobonOrientierungswertSozialeInfrastrukturRepository;
+import de.muenchen.isi.infrastructure.repository.stammdaten.StaedtebaulicheOrientierungswertRepository;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.fileupload.disk.DiskFileItem;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+@Component
+@Data
+@Slf4j
+public class ImportStammdatenFromFile implements CommandLineRunner {
+
+    private static final String PATH = "csv/";
```

Comment:
Pfad zum Ordner der CSV-Dateien als Property vorhalten und in Propertydatei pflegen.